lms/djangoapps/static_template_view/urls.py

```diff
@@ -52,6 +52,4 @@
 
     # Make the assumption that the URL we want is the lowercased
     # version of the map key
-    urlpatterns += (url(r'^%s$' % key.lower(), 'render', {'template': template}, name=value),)
-
-urlpatterns = patterns(*urlpatterns)
+    urlpatterns += [url(r'^%s$' % key.lower(), views.render, {'template': template}, name=value), ]
```

Comment:
Maybe just `append()` the new URL to the list directly?

Reply:
👍